app/src/main/res/layout/tab_fragment_main.xml

```diff
@@ -6,51 +6,10 @@
     android:orientation="vertical">
 
 
-    <TextView
```

Comment:
I'm assuming that now anywhere you are in the project this information is shown

Reply:
You're correct, I assumed that you put this area as a place holder. the 'Hello Tomer' should be in the logo area according to the mockup (we haven't implemented it yet), and the selection of the crust/size should be in the tabs' titles, and I implemented it in the pull request. As you said, it will appear anywhere in the project